new-client/src/plugins/LayerSwitcher/components/LayerGroupItem.js

```diff
@@ -582,6 +624,26 @@ class LayerGroupItem extends Component {
     );
   };
 
+  showZoomSnack(subLayer) {
+    if (this.zoomWarningSnack) return;
```

Comment:
Can you expand with some relevant comments? Here is a good place as it might at first seem counter-intuitive (why are we just leaving the method if `zoomWarningSnack` is truthy?).

Reply:
The method ensures that only one `Snackbar` notification is displayed at a time, preventing multiple notifications from overlapping.